src/MSBuildLocator/DiscoveryType.cs

```diff
@@ -25,6 +25,11 @@ public enum DiscoveryType
         /// <summary>
         ///     Discovery via dotnet --info.
         /// </summary>
-        DotNetSdk = 4
+        DotNetSdk = 4,
+
+        /// <summary>
+        /// Discovery via the Microsoft internal-only build environment CoreXT.
+        /// </summary>
+        CoreXT = 8
```

Comment:
```suggestion
        CoreXT = 8,
```

(this nit minimizes the next diff)